packages/showcase/src/user-controls/common-snippets/common-snippets.component.html

```diff
@@ -15,6 +15,52 @@
       </p>
     </div>
   }
+
+  @case ("spacing-body-components") {
+    <p class="{{ CSS.PARAGRAPH }}">
+      The following are the web components of mercury that adhere to the global
+      padding classes mentioned above:
```

Comment:
We must not mix concepts. Mercury only defines "Components", not web-components.

```suggestion
      The following are components that adhere to the global padding classes mentioned above:
```